### PR TITLE
Adds layout for Ergodox, colemak_code_friendly

### PR DESCRIPTION
**Pull request by Leszek111**
## Description

Thanks to the great ideas in other community layouts I have made my best one ever.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/community/ergodox/colemak_code_friendly/keymap.c [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+           M_POINER,    KC_4,    KC_5,     KC_6,     KC_0,     KC_TRNS,
+  KC_ENT,  M_NOT_EQL,   KC_1,    KC_2,     KC_3,     KC_TRNS,  KC_TRNS,
+                        KC_0,    KC_COMM,  KC_DOT,   KC_TRNS,  KC_TRNS,
+  KC_TRNS, DYN_REC_STOP,
+  DYN_REC_START2,
+  DYN_MACRO_PLAY2, KC_TRNS, KC_TRNS
+),
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (noroadsleft) [inline layouts/community/ergodox/colemak_code_friendly/keymap.c]: Lines 207 and 208 should be deleted; that function is deprecated, and your keymap doesn't use it anyway.

File: layouts/community/ergodox/colemak_code_friendly/keymap.c [hunk 2]
```diff
@@ -0,0 +1,328 @@
+/* -*- Mode:C; c-basic-offset:2; tab-width:2; indent-tabs-mode:nil; evil-indent-convert-tabs:t; -*- */
+
+#include QMK_KEYBOARD_H
+#include "debug.h"
+#include "action_layer.h"
```
[the remark this answers is not in the document]
Comment (Leszek111) [reply layouts/community/ergodox/colemak_code_friendly/keymap.c]: Removed, thanks!

File: layouts/community/ergodox/colemak_code_friendly/keymap.c (hunk 1 above)
Comment (Leszek111) [reply layouts/community/ergodox/colemak_code_friendly/keymap.c]: Removed, thanks!

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**